OrbitGl/TimerTrack.cpp

```diff
@@ -107,11 +107,22 @@ void TimerTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick,
 
         bool is_visible_width = normalized_length * canvas->GetWidth() > 1;
         bool is_selected = &text_box == GOrbitApp->selected_text_box();
+        bool need_highlight = GOrbitApp->selected_text_box() &&
+                              timer_info.thread_id() != SamplingProfiler::kAllThreadsFakeTid &&
+                              timer_info.thread_id() == GOrbitApp->selected_thread_id() &&
+                              timer_info.function_address() ==
+                                  GOrbitApp->selected_text_box()->GetTimerInfo().function_address();
+        need_highlight =
```

Comment:
The first statement is for the case that we pick a textbox in the capture window, and the second one is for the case that we pick a function from the live function data view panel. But this might not be a good coding style. Thanks for pointing out this. I will change to use parenthesis for distinguishing between those two cases.